src/main/java/ua/javaPro/berezhnoy/hillelHomeWork4/ArrayDataException.java

```diff
@@ -0,0 +1,7 @@
+package ua.javaPro.berezhnoy.hillelHomeWork4;
+
+public class ArrayDataException extends Exception {
```

Comment:
```suggestion
public class ArrayDataException extends RuntimeException {
```
Це робить наше життя простішим :)